README.md

```diff
@@ -16,6 +16,28 @@ Installation requires a minimum version (2.0.0) of the the Github CLI to support
 gh extension install dlvhdr/gh-prs
 ```
 
+> if you get an error when execute `gh prs`, do these steps:
+
```

Comment:
Looks good!
I would put it as the last section in the readme and add a title called "Troubleshooting"

Reply:
Or maybe a section called "Installing Manually"

Reply:
ok, I'll use `Installing Manually`